forms-thanks/ask-archives-mit-thanks.html

```diff
@@ -1,46 +1,98 @@
-<!doctype html>
-<html lang="en">
+<!DOCTYPE html>
+<!--[if lte IE 9]><html class="no-js lte-ie9" lang="en"><![endif]-->
+<!--[if !(IE 8) | !(IE 9) ]><!-->
+<html lang="en-US" class="no-js">
+<!--<![endif]-->
 <head>
+<meta charset="UTF-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
   <title>Thank you: Ask the Archives! Institute Archives &amp; Special Collections: MIT</title>
-  <link href="/css/header-semantic.css" rel="stylesheet" type="text/css">
-  <link href="/css/wp-libraries.css" rel="stylesheet" type="text/css">
-  <script type="text/javascript" src="../scripts/googleanalytics-async.js"></script>
+
+  <!--#include virtual="/mitlib-style/inc/assets-site-head.html" -->
+
+  <!-- form specific stuff -->
```

Comment:
We can remove all these styles since they aren't applicable to the thanks pages.